scripts/newinstall.sh

```diff
@@ -165,7 +165,7 @@ fi
 ##########	Test/warn about Python versions, offer to get miniconda2 if too old
 
 if true; then
-	PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
+	PYVEROK=$($PYTHON -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
```

Comment:
My interpretation of "otherwise specified" was that a different python will be used for the bootstrap if the `$PYTHON` environment variable is set. The problem is that `$PYTHON` is not the same as "Python to use to run/build software" because `newinstall.sh` uses `$PYTHON` (and can be overridden on the command line) but the subsequent software uses `python` from the PATH directly. This implies to me that the code was more correct before when `python` was used and this patch should not be applied.